core/win32/loader.c

```diff
@@ -1532,10 +1532,15 @@ map_api_set_dll(const char *name, privmod_t *dependent)
              str_case_prefix(name, "API-MS-Win-Eventing-Provider-L1-1"))
         return "kernelbase.dll";
     else if (str_case_prefix(name, "API-MS-Win-Core-PrivateProfile-L1-1") ||
-             str_case_prefix(name, "API-MS-Win-Core-Atoms-L1-1"))
+             str_case_prefix(name, "API-MS-Win-Core-Atoms-L1-1") ||
+             str_case_prefix(name, "API-MS-Win-Core-Localization-Obsolete-L1-2") ||
+             str_case_prefix(name, "API-MS-Win-Core-Kernel32-private-L1-1"))
```

Comment:
nit: to match the others: s/private/Private/

Reply:
can you make that change and merge please?

thanks